kontrakt/src/main/java/no/nav/abakus/iaygrunnlag/inntektsmelding/v1/RefusjonskravDatoDto.java

```diff
@@ -29,16 +29,21 @@
     @Valid
     private LocalDate førsteDagMedRefusjonskrav;
 
+    @JsonProperty(value = "harRefusjonFraStart", required = true)
```

Comment:
Her legger man inn ett nytt påkrevet felt i kontrakten. Hva skjer hvis konsumentene ikke implementerer feltet? Er endringen backwards kompatibel ?

Reply:
Kontrakten brukes kun til output.

Reply:
Det vil si at hvis jeg produserer en objekt A som output og så sender jeg plutselig objekt B vil det være fortsatt det samme for alle de som benytter seg av objektet A? 
Vil ikke si at det ikke vil virke her i dette tilfelle da vi fortsatt lever kun med fpsak som vi selv fikser men man må være obs i fremtiden når man får flere konsumente.

Reply:
Det vil fortsatt funke for alle som benytter seg av objekt A om objekt B fortsatt inneholder alle feltene til objekt A (har testet lokalt).

Reply:
Det hva jeg vil påpeke er at vi må kommunisere godt hvis man gjennomfører endringer i kontrakten. En så lenge holder vi på med fpsak og abakus men senere vil vi ikke ha så stor kontroll over konsumentene.
Så vi må bare:
- Være obs på at endringene vi gjør en backwards kompatible
- Kommunisere godt endringen gjennom våre offisielle kanaler godt tid før.